app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = Post.all.order(id: :desc)
+    @latest_posts = Post.all.order(id: :desc).limit(5)
```

Comment:
- [ ] Same here, Please remove this variable because you suppose to use `latest_comments` method that you created in the `post` model in the `post index view`.

https://github.com/topeogunleye/blog-app-2.0/blob/39b9f3a31aff529bd0e1945b1b45b8a28c20fa44/app/models/post.rb#L16-L18